### ISSUE TITLE
Build pipeline is partly broken

### ISSUE THREAD

**Issue by yp87** [yp87/leaf2mqtt#41]
Hello @EtienneMD, It looks like the build is partly failing because of the `Home Assistant addon` part of the build pipeline.
https://github.com/yp87/leaf2mqtt/actions/runs/3293147769/jobs/5429374074

Thank you!

**Comment by EtienneMD** [comment]
Are you sure your branch was up to date with Main when you pushed? Commit edcf6184441f086dc160436d2c4bea6d85083085 shows 7c9aeceeab034e00059b249c4933f03fd11c9ea9 as parent which dates back 4 months ago. Since origin and remote were different it seems to have caused a conflict.

**Comment by EtienneMD** [comment]
I might be wrong thought. I will update the addon manually and we will see on the next update. 
Feel free to keep this issue open.

**Comment by yp87** [comment]
The commit where I resolved the conflict has both has parents (the previous commit which skipped previous 2 commits and the last commit from september), So I think its good.
![image](https://user-images.githubusercontent.com/4250335/197083815-edb0f9ea-fd39-4be3-ace1-c856c9111873.png)

So juste read the logs again and what happened is that I merged an unrelated PR after the pipeline checked out the code, but before it tried to push to master to update the file.
So the push failed.

I created a new run and now it worked. Thank you!
